### DISCUSSION TITLE
Possible bot refactor?

### DISCUSSION THREAD

**Discussion by red40forever** [Refloow/Steam-Card-Bot-PRO#34] [General]
After looking through the bot's code for a while, it seems to me that it's in dire need of restructuring. If there is significant interest, I  will open an issue, thank you.

**Comment by Refloow** [comment]
> After looking through the bot's code for a while, it seems to me that it's in dire need of restructuring. If there is significant interest, I will open an issue, thank you.

Code is well documented with comments but it is long and would need some restructuring at places. If you want to contribute read [contribution guide](https://github.com/OSL-Works/Steam-Card-Bot-PRO/blob/master/.github/CONTRIBUTING.md) and feel free to create an pull request :D 

I'm working on new update, i push updates based on activity of projects so feel free to leave an star.
